web-client/src/view/BloodCenters/index.tsx

```diff
@@ -141,14 +148,22 @@ const BloodCenters: React.FC = observer(() => {
         </Box>
         <Text fontSize='24px' fontWeight='600'>Blood center nearby</Text>
         <Text color="#57575b" fontSize="16px">Since your geolocation is enabled, the nearest blood centers are shown.</Text>
-        <ul>
-          {bloodCenters.map((center: IBloodDonationCenter, index: number) => (
-            < li key={center.id} >
-              <strong>{index + 1}.{center.name}</strong>
-            </li>
+        <CardContainer>
+          {bloodCenters.map((center: IBloodDonationCenter) => (
+            <Card key={center.id} onClick={()=>navigate(`center/${center.id}`)} cursor="pointer">
+              <FlexBox align="center">
+                <LocationIcon />
+                <FlexBox direction="column"  margin="0 0 0 16px">
+                  <Text color="var(--header-text-color)" fontWeight="600" cursor="pointer">{center.name}</Text>
+                  <Text color="var(--header-text-color)" cursor="pointer">{center?.address?.street}, {center?.address?.city}</Text>
+                  <Text color="var(--header-text-color)" cursor="pointer">{center?.address?.postalCode}, {center?.address?.region}</Text>
+                </FlexBox>
+              </FlexBox>
+              <Badge>{transformFirstLetter(center.category)}</Badge>
```

Comment:
Would css text-transform: capitalize; also work?

Reply:
We receive all the letters capitalized, so I needed to create this helper. But I thank you for this question)! I found another solution:
  text-transform: lowercase;
  &:first-letter {
    text-transform: uppercase;
  }